express-notes-api/index.js

```diff
@@ -0,0 +1,87 @@
+const data = require('./data');
+const fs = require('fs');
+
+var express = require('express');
+var app = express();
+
+app.use(express.json());
+
+app.get('/api/notes', (req, res) => {
+  const dataArray = [];
+  for (const key in data.notes) {
+    dataArray.push(data.notes[key]);
+  }
+  res.json(dataArray);
+});
+
+app.get('/api/notes/:id', (req, res) => {
+  // const numID = Number(req.params.id);
+  if (!Number.isInteger(req.params.id) && req.params.id < 0) {
+    res.status(400).json({ error: 'id must be a positive integer' });
```

Comment:
An id of `0` should result in a `400` error (zero isn't positive, it's just zero).